Detectors/Calibration/workflow/ccdb-populator-workflow.cxx

```diff
@@ -33,7 +33,8 @@ void customize(std::vector<o2::framework::CompletionPolicy>& policies)
   // we customize the pipeline processors to consume data as it comes
   using CompletionPolicy = o2::framework::CompletionPolicy;
   using CompletionPolicyHelpers = o2::framework::CompletionPolicyHelpers;
-  policies.push_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
+  auto pol = policies.push_back(CompletionPolicyHelpers::defineByName("ccdb-populator.*", CompletionPolicy::CompletionOp::Consume));
+  pol.order = CompletionPolicy::CompletionOrder::Timeslice;
```

Comment:
Actually I realised we need a "Slot" ordering: 

```suggestion
  pol.order = CompletionPolicy::CompletionOrder::Slot;
```

because `Timeslices` are always the same for things created inside a given iteration of the data processor, so ordering by them does not help. This would still not work correctly in case slots are filled in non time ordered manner, but I think for your usecase it should not happen.

A better way would be to record the order of arrival of the messages, and sort by it, however that requires a few more modifications. I suggest we go ahead with this and iterate after I am back on the 7th.